src/main/java/racinggame/domain/RacingGame.java

```diff
@@ -0,0 +1,54 @@
+package racinggame.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RacingGame {
+    private static List<Car> cars = new ArrayList<>();
```

Comment:
```suggestion
    private final List<Car> cars;
```
상태 값을 가지는 인스턴스 변수는 static으로 구현할 경우 멀티 쓰레드 환경에서 문제가 발생할 수 있음.
특별한 이유가 없는 한 위와 같이 인스턴스 변수로 구현할 것을 추천

이번 기회에 클래스와 인스턴스의 차이점에 대해 추가 학습하는 시간으로 만들어볼 것을 추천